.github/workflows/publish_pytorch_dev_docker.yml

```diff
@@ -34,35 +39,43 @@ jobs:
           git config fetch.parallel 10
           git config --global user.email "nobody@amd.com"
           git config --global user.name "Nobody"
+
+      - name: Install dependencies
+        run: sudo apt-get update && sudo apt-get install -y uuid-runtime gh
+
+      - name: Generate unique dispatch ID
+        run: echo "DISPATCH_ID=dispatch-${{ matrix.targets.amdgpu_target }}-$(uuidgen)" >> $GITHUB_ENV
```

Comment:
What about using `github.run_id` or ``${{ matrix.targets.amdgpu_target }}-${{ github.run_id }}`?

Reply:
Going through the changes further, is there any real use of the dispatch ID beside in the bash script?

Reply:
I can try `github.run_id` as well